### PR TITLE
Clickhouse m1 support

### LINKED ISSUE #3990: Enable M1 development with docker
According to the onboarding docs new hires are to buy an M1 mac as their laptop (if they go the apple route). Currently our stack doesn't fully support M1 and this has slowed down the onboarding of new hires. We should sort out the story for dev on M1.

Ideal solution:
- Docker runs entire stack locally

Alternate solution:
- Use dedicated or shared dev box to develop for clickhouse, kafka, and zookeeper.

### PR DESCRIPTION
**Pull request by buwilliams**
Clickhouse docker does not work out of the box for Apple M1. Closes #3990 

## Changes

- New `clickhouse.arm64.Dockerfile` to build Clickhouse images/containers correctly for local development
- New `docker-compose.arm.yml` to support arm64/M1
- New bin scripts `start-arm` `stop-arm` to showcase how to run docker compose
- New `config-clickhouse.arm64.xml` for Clickhouse for arm64

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

File: posthog/settings.py [hunk 1]
```diff
@@ -167,6 +167,14 @@ def print_warning(warning_lines: Sequence[str]):
 KAFKA_HOSTS = ",".join(KAFKA_HOSTS_LIST)
 KAFKA_BASE64_KEYS = get_from_env("KAFKA_BASE64_KEYS", False, type_cast=strtobool)
 
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
+print("KAFKA_URL: " + KAFKA_URL)
```
Comment (fuziontech) [inline posthog/settings.py:176]: printer go brrrrrrrrr

Comment (buwilliams) [reply posthog/settings.py:176]: This will be fixed before it goes out I promise. PR wasn't meant to be merged.

File: docker-compose.arm64.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            dockerfile: clickhouse.arm64.Dockerfile
+        command: clickhouse-server -C /etc/clickhouse-server/config.xml
+        volumes:
+            - /var/lib/clickhouse/
+        ports:
+            - '8123:8123'
+            - '9000:9000'
+            - '9440:9440'
+            - '9009:9009'
+            - '8000:8000'
```
Comment (fuziontech) [inline docker-compose.arm64.yml:63]: you have this defined twice which will break the compose config

File: docker-compose.arm64.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    zookeeper:
+        image: wurstmeister/zookeeper
+    kafka:
+        image: wurstmeister/kafka
+        depends_on:
+            - zookeeper
+        ports:
+            - '9092:9092'
+        environment:
+            KAFKA_ADVERTISED_HOST_NAME: localhost
```
Comment (fuziontech) [inline docker-compose.arm64.yml:75]: this needs to be `kafka`

File: docker-compose.arm64.yml (hunk 2 above)
Comment (buwilliams) [reply docker-compose.arm64.yml:63]: yeah, that was for networking testing, it's gone now

**Comment by buwilliams** [comment]
Closing this PR for now since M1 support isn't available.

**Pull request closed without merging**